### PR TITLE
Update CODEOWNERS

### PR DESCRIPTION
**Pull request by mihaitabamburic**

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -1 +1 @@
-@e-conomic/platform
```
Comment (Sarah-van-Dam) [inline .github/CODEOWNERS:1]: Why is the * needed?

Comment (mihaitabamburic) [reply .github/CODEOWNERS:1]: Because we want to get added as reviewers on any file changed

Comment (Dzoge) [reply .github/CODEOWNERS:1]: I have the same question too

Comment (Dzoge) [reply .github/CODEOWNERS:1]: @mihaitabamburic 
Then why are we removing `*` in other reps?

**Review by Sarah-van-Dam — APPROVED** [review]

**Pull request merged**